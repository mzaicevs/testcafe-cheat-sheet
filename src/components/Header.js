import React from "react";
import { Jumbotron, Container, Row, Image } from "react-bootstrap";

const Header = () => (
  <Jumbotron>
    <Container style={{ marginLeft: "12px" }}>
      <Row>
        <a
          href="https://devexpress.github.io/testcafe/"
          target="_blank"
          rel="noopener noreferrer"
        >
          <Image
            src="https://devexpress.github.io/testcafe/images/site-header-logo.svg"
            rounded
          />
        </a>
      </Row>
      <Row>
        <h1>Cheat Sheet</h1>
      </Row>
      <Row>
        <p>Copy TestCafe commands and code...and stuff!</p>
      </Row>
    </Container>
  </Jumbotron>
);

export default Header;
