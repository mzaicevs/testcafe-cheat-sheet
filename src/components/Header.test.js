import React from "react";
import { shallow } from "enzyme";
import Header from "./Header";

describe("Header", () => {
  it("matches snapshot", () => {
    const wrapper = shallow(<Header />);
    expect(wrapper).toMatchSnapshot();
  });
});
