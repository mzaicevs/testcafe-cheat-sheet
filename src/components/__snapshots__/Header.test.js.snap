// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header matches snapshot 1`] = `
<Bootstrap(Jumbotron)>
  <Container
    as="div"
    fluid={false}
    style={
      Object {
        "marginLeft": "12px",
      }
    }
  >
    <Bootstrap(Row)>
      <a
        href="https://devexpress.github.io/testcafe/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <Bootstrap(Image)
          rounded={true}
          src="https://devexpress.github.io/testcafe/images/site-header-logo.svg"
        />
      </a>
    </Bootstrap(Row)>
    <Bootstrap(Row)>
      <h1>
        Cheat Sheet
      </h1>
    </Bootstrap(Row)>
    <Bootstrap(Row)>
      <p>
        Copy TestCafe commands and code and stuff!
      </p>
    </Bootstrap(Row)>
  </Container>
</Bootstrap(Jumbotron)>
`;
